test/functional/desired.js

```diff
@@ -16,8 +17,13 @@ const APIDEMO_CAPS = Object.assign({}, GENERIC_CAPS, {
   app: apidemosApp,
 });
 
+const COMPOSE_CAPS = Object.assign({}, GENERIC_CAPS, {
+  app: path.resolve('test', 'assets', 'compose_playground.apk'),
+  espressoBuildConfig: '{"additionalAndroidTestDependencies": ["androidx.lifecycle:lifecycle-extensions:2.2.0", "androidx.activity:activity:1.3.1", "androidx.fragment:fragment:1.3.4"]}'
```

Comment:
please split this long line into multiple

Reply:
done